CHANGELOG.md

```diff
@@ -10,6 +10,13 @@
 
 ## 2.28.3
 
+### New features
```

Comment:
This should go under the `Unreleased` section above, not in `2.83.3`

Reply:
Good catch, addressed.

Reply:
Good catch, addressed.